performance_tests/10zone1vav.idf

```diff
@@ -35,10 +35,12 @@
   Output:DebuggingData,0,0;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
+    1.,                      !- Name
```

Comment:
min-fields isn't the issue here.  The files that are showing the diffs in ResearchSpecial weren't properly transitioned to v8.7  Name and Zone Name were new fields in v8.7.  Looking at 20zone1VAV, for example, in develop, the file says v8.7, but the new fields are not present:
```
  ZoneCapacitanceMultiplier:ResearchSpecial,
    1.,                      !- Temperature Capacity Multiplier
    1.,                      !- Humidity Capacity Multiplier
    1.,                      !- Carbon Dioxide Capacity Multiplier
    1.0;                     !- Generic Contaminant Capacity Multiplier
```
Probably need to go back and re-transition the performance test files from 8.6 to 8.7.
And why isn't ZoneCapacitanceMultiplier:ResearchSpecial throwing an error for Zone Name = "1."?